src/level_set/FESurfaceDistanceEvaluator.cpp

```diff
@@ -106,25 +109,22 @@ line_equation(const IBTK::Vector3d& coord, const libMesh::Point& n0, const libMe
     return (y1 - y0) * coord(0) + (x0 - x1) * coord(1) + (x1 * y0 - x0 * y1);
 } // line_equation
 
+#define MAKE_EDGE(n0, n1) n1(0) - n0(0), n1(1) - n0(1), n1(2) - n0(2)
```

Comment:
Why is this a macro? It would make more sense for this to be an inline function in an anonymous namespace.

Reply:
I make use of macro expansion for both IBTK::Vector3d and the libMesh::VectorValue. 

''' const IBTK::Vector3d E0(MAKE_EDGE(B, n1)); '''

''' Edge[0] = VectorValue<double>(MAKE_EDGE(n0, n1)); '''

For this, I have to write two separate inline functions.

Reply:
I use single MAKE_EDGE macro for IBTK::Vector3d, libMesh::VectorValue and libMesh::Point.

Code snippet is here:
```
const IBTK::Vector3d E0(MAKE_EDGE(B, n1));
Edge[0] = VectorValue<double>(MAKE_EDGE(n0, n1));
nbr_e0 << MAKE_EDGE(nbr_n0, nbr_n1); //nbr_n0 and nbr_n1 are libmesh::Point
```
In this case, I may have to write separate inline functions.

Reply:
Macro is fine.

Reply:
I would prefer to still use a template function rather than this but its not a big deal; lets leave it if @amneetb is happy.

Reply:
We should prefer using functions instead of macros. In this case a templated function should work and is not much more code.

Reply:
I replaced the macro with the function template and updated the code.